content/help/ops/traffic-management/locality-load-balancing/index.md

```diff
@@ -50,11 +54,42 @@ The hierarchy of prioritization matches in the following order:
 
 Proxies in the same zone but different regions are not considered local to one another.
 
+In order for Envoy to determine when instances are unhealthy, an [Outlier Detection](/docs/reference/config/networking/v1alpha3/destination-rule/#OutlierDetection)
+rule must be defined for each service.
+
 ### Overriding the Locality Fail-over
 
 Sometimes, you need to constrain the traffic fail-over to avoid sending traffic to
 endpoints across the globe when there are not enough healthy endpoints in the
 same region. This behavior is useful when sending fail-over traffic across regions
 would not improve service health or many other reasons including regulatory controls.
-To constrain traffic to a region, use the mesh `LocalityLoadBalancerSetting.Failover`
-configuration detailed in the [Locality load balancing reference guide](/docs/reference/config/istio.mesh.v1alpha1/#LocalityLoadBalancerSetting).
+To constrain traffic to a region, configure the `localityLbSetting` option using Helm. See the
+[Locality load balancing reference guide](/docs/reference/config/istio.mesh.v1alpha1/#LocalityLoadBalancerSetting)
+for options.
+
+An example configuration:
+
+```yaml
+global:
+  localityLbSetting:
+    failover:
+    - from: us-east
+      to: eu-west
+    - from: us-west
+      to: us-east
+```
+
+## Locality-Weighted Load Balancing
```

Comment:
Only capitalize first character in headings:

Locality-weighted load balancing